tests/integration/components/delete-node-modal/component-test.ts

```diff
@@ -15,11 +15,6 @@ module('Integration | Component | delete-node-modal', hooks => {
         this.set('closeModal', () => true);
     });
 
-    test('hidden by default', async function(assert) {
-        await render(hbs`{{delete-node-modal closeModal=closeModal delete=delete}}`);
```

Comment:
It seems like BsModal's default behavior is to [show the modal now unless `@open` is explicitly set to `false`](https://github.com/ember-bootstrap/ember-bootstrap/blob/v4.6.3/addon/components/bs-modal.js#L674)

Reply:
It's probably not worth doing, but we could enforce the "hidden by default" behavior in the DeleteNodeModal if we wanted to.

Reply:
Hmmmm, the component relies on a passed in value to determine whether it's open, so I don't know if specifying a default state is appropriate anymore. Although I don't like the fact that BsModal is open for falsey values of `@open`... I have quite mild feelings going both ways on this one ⚖️

Reply:
I was thinking making the components `@open = {{#if @openModal true false}}` or something like that, which should coerce falsy values to false?

Reply:
Oh, yea, that didn't cross my mind as something we can do, but that's simple enough! Will make those changes now